examples/image_resources/image_resources.py

```diff
@@ -57,7 +57,7 @@ async def on_message(event: hikari.GuildMessageCreateEvent) -> None:
 async def inspect_image(event: hikari.GuildMessageCreateEvent, what: str) -> None:
     """Inspect the image and respond to the user."""
     # Show the avatar for the given user ID:
-    if user_match := re.match(r"<@!?(\d+)>", what):
+    if (user_match := re.match(r"<@!?(\d+)>", what)):
```

Comment:
We follow the black code style, which would remove the brackets (as you can see in the linting workflow), so this can be reverted